pkg/asset/installconfig/baremetal/baremetal.go

```diff
@@ -26,8 +26,25 @@ func Platform() (*baremetal.Platform, error) {
 		return nil, err
 	}
 
+	// FIXME: bare metal - use net.IP type
+	var masterVIP string
+	err = survey.Ask([]*survey.Question{
```

Comment:
The philosophy of the openshift installer is to reduce the amount of survey questions as much as possible.

Since one of the requirements we impose on the admins is that they set up a DNS record for api.clustername.basedomain. I think the best to do here is to get it using the net library like in the example below:

```
package main

import (
        "net"
        "fmt"
        "os"
)

func main() {
        domain := "api.ostest.test.metalkube.org"
        ips, err := net.LookupIP(domain)
        if err != nil {
                fmt.Fprintf(os.Stderr, "Could not get IPs: %v\n", err)
                os.Exit(1)
        }
        for _, ip := range ips {
                fmt.Printf("%s IN A %s\n", domain, ip.String())
        }
}```

Reply:
After discussing it with @russellb. It's better not to do what I propose so that we can work in environments where the DNS record points to a preNATted address.